### PR TITLE
Ticket2994 keithley 2700 display temp and drift

### PR DESCRIPTION
**Pull request by esouthren**

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py [hunk 1]
```diff
@@ -12,11 +12,18 @@
 table = widget.getTable()
 row = 1
 color = ColorFontUtil.getColorFromRGB(240,245,245)
+
 i = 0
 for i in range(20):
-    reading = PVUtil.getString(pvs[i]) 
+    reading = PVUtil.getString(pvs[i])
+    temperature = PVUtil.getString(pvs[i+20])
```
Comment (AdrianPotter) [inline base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: Beware of magic numbers. Is this `20` the same as the `20` in `for i in range(20):`? I guess so, but it's not obvious. It's better to have.

```
offset_between_temperature_and_drift_pv_indexes = 20
...
```

It's maybe a tad verbose, but something along those lines rould remove confusion

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py [hunk 2]
```diff
@@ -12,11 +12,18 @@
...
+    drift = PVUtil.getString(pvs[i+40])
```
Comment (AdrianPotter) [inline base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: 2nd magic number. Is this `2*offset_between_temperature_and_drift_pv_indexes` or something else? I can't know without examining the entire code, reducing overall readability

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py [hunk 3]
```diff
@@ -12,11 +12,18 @@
...
+     
     table.setCellText(row, 3, reading)
+    table.setCellText(row, 1, temperature)
+    table.setCellText(row, 2, drift)
```
Comment (AdrianPotter) [inline base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: Although these numbers aren't repeated, it's good practice to have variables (preferably `static` and  `final` if we were in Java land)  `reading_column_index` etc. It makes future changes easier and maintains clarity if those variables end up being reused elsewhere.

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py (hunk 1 above)
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: Removed magic numbers

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py (hunk 2 above)
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: Removed

**Pull request merged**